packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -45,6 +46,18 @@ const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline', {
 });
 ```
 
+If key rotation is required for KMS keys, you can configure it by passing
+`enableKeyRotation: true` when defining the pipeline. Ensure that
+`crossAccountKeys: true` is also passed. Enabling key rotation will
+incur an additional cost of **$1/month**.
+
+```ts
+const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline', {
+  crossAccountKeys: true,
+  enableKeyRotation: true,
+});
+```
```

Comment:
````suggestion
If you want to enable key rotation for the generated for KMS keys,
you can configure it by passing `enableKeyRotation: true` when creating the pipeline.
Note that key rotation will incur an additional cost of **$1/month**.

```ts
const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline', {
  // ...
  enableKeyRotation: true,
});
```
````